certbot-dns-route53/certbot_dns_route53/dns_route53_test.py

```diff
@@ -1,4 +1,4 @@
-"""Tests for certbot_route53.authenticator"""
+"""Tests for certbot_dns_route53.authenticator"""
```

Comment:
I think this should be `certbot_dns_route53.dns_route53` or `certbot_dns_route53.dns_route53.Authenticator`.